docs/source/configuration_file.md

```diff
@@ -0,0 +1,159 @@
+# Configuration file
+
+## General parameters
+
+#### command
```

Comment:
i would follow the logical headings order (##, then ###, then ####), otherwise it will impact auto-TOC generation
```suggestion
### command
```